### PR TITLE
feat(deprec): move createClass/createElement functions under CMS object

### PR DESCRIPTION
**Pull request by tech4him1**
**- Summary**

See discussion in #588.

**- Test plan**

**- Description for the changelog**

Deprecations always require a minor release, not just a patch, so this is labeled `feat(deprec)`. Maybe there is a better way?

**- A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

**Comment by verythorough** [comment]
Deploy preview for *cms-demo* ready!

Built with commit 783881335fe9d90a58e229964432afaf5ef3658e

https://deploy-preview-1509--cms-demo.netlify.com

**Comment by verythorough** [comment]
Deploy preview for *netlify-cms-www* ready!

Built with commit 783881335fe9d90a58e229964432afaf5ef3658e

https://deploy-preview-1509--netlify-cms-www.netlify.com

File: example/index.html [hunk 1]
```diff
@@ -92,6 +92,9 @@
 
   <script src='/cms.js'></script>
   <script>
+    var createClass = CMS.createClass;
+    var h = CMS.h;
```
Comment (tech4him1) [inline example/index.html]: I'm not sure if aliasing like this as an example in the docs is a good idea or not. I've done it in the "Customization" doc, but not the "Custom Widgets" doc. Any thoughts?

**Comment by erquhart** [comment]
We should discuss these changes as a part of the larger conversation around shared dependencies.

**Comment by erquhart** [comment]
@tech4him1 okay to close this for now and handle under a higher level effort? Want to avoid moving them twice.

**Pull request closed without merging**